SaltedgeAuthenticatorCore/Classes/API/SEEncryptedData.swift

```diff
@@ -22,7 +22,7 @@
 
 import Foundation
 
-public struct SEEncryptedData: SEBaseEncryptedAuthorizationData, SerializableResponse, Equatable {
+public struct SEEncryptedData: SEBaseEncryptedAuthorizationData, Decodable, Equatable {
```

Comment:
`SerializableResponse` file should be deleted from project after migration to Codable